common/protos/identity/agent.proto

```diff
@@ -29,30 +29,31 @@ message SecurityDomainMemberIdentity {
 }
 
 message SecurityDomainMemberIdentityRequest {
-    SecurityDomainMemberIdentity source_network = 1;
-    SecurityDomainMemberIdentity requesting_network = 2;
-    string nonce = 3;
+  SecurityDomainMemberIdentity source_network = 1;
+  SecurityDomainMemberIdentity requesting_network = 2;
+  string nonce = 3;
 }
 
 // Association of signature (over arbitrary data) and signer identity
 message Attestation {
   SecurityDomainMemberIdentity unit_identity = 1;
   string certificate = 2;
   string signature = 3;
+  string nonce = 4;
 }
 
 // Attested security domain membership by a single member
 message AttestedMembership {
-  common.membership.Membership membership = 1;
+  bytes membership = 1;         // 'common.membership.Membership': Base64-encoded
```

Comment:
If its base64 encoded, shouldn't it be a string?

Reply:
Yeah, I thought about this while upgrading the CLI. Will change to `string`.